panoptes_aggregation/routes.py

```diff
@@ -69,6 +70,15 @@ def make_application():
                         static_url_path='',
                         static_folder='../docs/build/html')
     application.json_encoder = MyEncoder
+    CORS(
+        application,
+        origins=[
+            r'^https?:\/\/([a-z0-9-.]+zooniverse.org)'
```

Comment:
Is it possible to add localhost here for development purposes?

Reply:
Sure, I assume a regex similar to this one form Panoptes will work https://github.com/zooniverse/Panoptes/blob/master/config/cors_config.yml.hudson#L7

Reply:
I'd prefer we use `local.zooniverse.org` for local dev.